vendor/mooltipass/mcCombinations.js

```diff
@@ -1020,6 +1020,7 @@ mcCombinations.prototype.getAllForms = function () {
 
         // Ignore our fields and search fields.
         if (field.attr('id') == 'mooltipass-password-generator' ||
+            field.attr('id') == 'inputWRSL301_rechercheTexte' ||
```

Comment:
could you make it more generic and just search for "recherche" within the id field?